transporter.js

```diff
@@ -0,0 +1,26 @@
+const nodemailer = require('nodemailer');
+const cred = require('./emailConfig.json');
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module './emailConfig.json'.